kaptan/__init__.py

```diff
@@ -52,20 +53,39 @@ def upsert(self, key, value):
         self.configuration_data.update({key: value})
         return self
 
+    def _is_python_file(self, value):
+        ext = op.splitext(value)[1][1:]
+        if ext == 'py' or op.isfile(value + '.py'):
+            return True
+        return False
+
     def import_config(self, value):
-        if not isinstance(value, dict) and os.path.isfile(value):
+        if isinstance(value, dict):  # load python dict
+            self.handler = self.HANDLER_MAP['dict']()
+            data = value
+        elif op.isfile(value) and not self._is_python_file(value):
             if not self.handler:
                 try:
-                    key = HANDLER_EXT.get(os.path.splitext(value)[1][1:], None)
+                    key = HANDLER_EXT.get(op.splitext(value)[1][1:], None)
```

Comment:
Just keeping it as ``os.path`` is a bit more readable.  It's not immediately obvious what ``op`` is if someone is glancing through it (unless they remembered how ``os.path`` was imported as ``op`` above).  May as well make it simpler for them :smile: